recipes/scikit-rf/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.14.4" %}
+
+package:
+  name: scikit-rf
+  version: {{ version }}
+
+source:
+  fn: scikit-rf-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/scikit-rf/scikit-rf-{{ version }}.tar.gz
+  sha256: c7643feb5142670d100515ec160e8b8644120b6c99a76e685aba4a90f0eb93cf
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - pandas
+    - ipython
+    - numpy
+    - scipy
+    - matplotlib
+    - six
+    - future
+
+test:
+  imports:
+    - skrf
+    - skrf.calibration
+    - skrf.data
+    - skrf.io
+    - skrf.media
+
+about:
+  home: http://www.scikit-rf.org
+  license: BSD 3-Clause
```

Comment:
ok, fyi the scikit-rf licence is in [skrf/LICENSE.txt ](https://github.com/scikit-rf/scikit-rf/blob/master/LICENSE.txt)